channeldb/db.go

```diff
@@ -318,19 +278,12 @@ func createChannelDB(dbPath string) error {
 			return err
 		}
 
-		graphMeta, err := tx.CreateTopLevelBucket(graphMetaBucket)
-		if err != nil {
-			return err
-		}
+		graphMeta := tx.ReadWriteBucket(graphMetaBucket)
```

Comment:
Go all the way and describe the initial bucket hierarchy in a tree-like datastructure?

Reply:
seems tangential to this PR? only reason i did this for TLBs is because i would be duplicating code in adding `CreateTopLevelBucket` and `DeleteTopLevelBucket` for the pay addr index.

Reply:
Yes, absolutely scope creep